adapters/arpalazio.js

```diff
@@ -0,0 +1,298 @@
+'use strict';
+
+import { REQUEST_TIMEOUT } from '../lib/constants';
+import { default as baseRequest } from 'request';
+const request = baseRequest.defaults({timeout: REQUEST_TIMEOUT});
+import { default as moment } from 'moment-timezone';
+import cheerio from 'cheerio';
+import { parallel } from 'async';
+import { acceptableParameters, convertUnits } from '../lib/utils';
+import { default as parse } from 'csv-parse/lib/sync';
+import { difference, flattenDeep, zip } from 'lodash';
+
+exports.name = 'arpalazio';
+
+const baseUrl = 'http://www.arpalazio.net/main/aria/sci/annoincorso/';
+const provinceQueryPath = 'chimici/chimici.php';
+const hourlyAvgParam = 0;
+const dailyAvgParam = 3;
+const hourlyAvgPeriod = {unit: 'hours', value: 1};
+const dailyAvgPeriod = {unit: 'hours', value: 24};
+const dailyParameters = ['pm25', 'pm10'];
```

Comment:
We've talked about it on #282 (starting at [comment](https://github.com/openaq/openaq-fetch/issues/282#issuecomment-317225793)).